src/styles/index.scss

```diff
@@ -1,3 +1,120 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,100&display=swap';
+
+* {
```

Comment:
According to the code style rules, there should not be a space before the opening brace '{'.